### PR TITLE
Release notes MTA 5.1.3

### PR DESCRIPTION
**Pull request by RichardHoch**
Release notes for MTA 5.1.3
Resolves: https://issues.redhat.com/browse/WINDUP-3025
Preview: http://file.emea.redhat.com/rhoch/10032021/513rn/html-single/

### CODE REVIEW

**Comment by RichardHoch** [comment]
@apinnick; please review.

File: docs/release-notes/release_notes-5.1.3/master.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+:imagesdir: topics/images
+:context: release-notes
+:release-notes:
+
+= {ReleaseNotesName}
+
+//Inclusive language statement
+include::topics/making-open-source-more-inclusive.adoc[]
+
+include::topics/rn-new-features-513.adoc[leveloffset=+1]
```
Comment (apinnick) [inline docs/release-notes/release_notes-5.1.3/master.adoc:14]: Why are you creating new files for 5.1.3? In the past, we reused the files.

Comment (apinnick) [reply docs/release-notes/release_notes-5.1.3/master.adoc:14]: I see what you're doing. Please don't create more folders. Just update the existing 5.1 release notes.

**Comment by RichardHoch** [comment]
@apinnick: Extra folders and files removed. Please review.

**Comment by apinnick** [comment]
LGTM. I am merging this PR but I will delay publishing the updated doc until the GA date.

**Pull request merged**